src/unittest/Dasm.cc

```diff
@@ -0,0 +1,55 @@
+#include "Dasm.hh"
+#include "EmuTime.hh"
+#include "MSXCPUInterface.hh"
+
+#include "catch.hpp"
+
+#include <iostream>
+
+using namespace openmsx;
+
+TEST_CASE("instructionLength, dasm")
+{
+	std::array<uint8_t, 4> opcode = {0, 0, 0, 0};
+	std::string dest;
+
+	bool end = false;
+	while (!end) {
+		auto len = instructionLength(opcode).value_or(0);
+		assert(1 <= len);
+		assert(len <= 4);
```

Comment:
In the context of `catch2` it might be better to replace `assert` with `REQUIRE`.